app/src/test/java/io/gnosis/safe/utils/SemVerTest.kt

```diff
@@ -18,6 +17,20 @@ class SemVerTest {
         assertEquals(SemVer(2, 15, 0, "255rc-internal"), version3)
     }
 
+    @Test
+    fun `parse (versionString, ignoreExtensions) should return SemVer`() {
+        val version1 = SemVer.parse("2.15.0", true)
+        assertEquals(SemVer(2, 15, 0), version1)
+
+        val version2 = SemVer.parse("2.15.0-255-internal", true)
+        assertNotEquals(SemVer(2, 15, 0, "255-internal"), version2)
```

Comment:
Why does this have to be not equal?

Reply:
yes. Here we test that extension part is ignored during parsing